### PR TITLE
Adding repository url metadata

### PR DESCRIPTION
**Pull request by MaximRouiller**

### CODE REVIEW

File: src/extension/nunit.v2.driver.csproj [hunk 1]
```diff
@@ -11,6 +11,8 @@
     <AssemblyName>nunit.v2.driver</AssemblyName>
     <TargetFrameworkVersion>v2.0</TargetFrameworkVersion>
     <FileAlignment>512</FileAlignment>
+    <RepositoryUrl>https://github.com/nunit-v2-framework-driver</RepositoryUrl>
+    <RepositoryType>git</RepositoryType>
```
Comment (CharliePoole) [inline src/extension/nunit.v2.driver.csproj:15]: Since the project file isn't used to create the nuget package, this doesn't seem to serve any purpose.

**Comment by CharliePoole** [comment]
In this project, you need to modify build.cake.

Various repos are authored by different people, so they may handle things differently.

File: src/extension/nunit.v2.driver.csproj (hunk 1 above)
Comment (MaximRouiller) [reply src/extension/nunit.v2.driver.csproj:15]: If it's not then I'll cancel this PR.

**Comment by MaximRouiller** [comment]
I have 0 experience with cake and will leave someone the joy of modifying it. ☹

Sorry I couldn't fix this one.

**Pull request closed without merging**